modules/social_features/social_post/templates/post.html.twig

```diff
@@ -17,11 +17,21 @@
 #}
 <div{{ attributes.addClass('card', 'post') }}>
 
-  {% if content.user_id %}
-    {{- content.user_id -}}
-  {% endif %}
-
-  {{ date }}
+  <div class="media">
+    <div class="media-left avatar">
+      {% if author_picture %}
+        {{ author_picture }}
+      {% endif %}
```

Comment:
@johnreytanquinco - what do we do if they dont have a profile pic?

Reply:
With this code added in that file, we have also enabled the default profile photo from OpenSocial.